### ISSUE TITLE
README Update/ Tutorial on How to Run Program

### ISSUE THREAD

**Issue by dejazzhands** [zillur-av/wrong-way-vehicle-detection#2]
I read your research paper and I was really impressed with your work. I am trying to train an AI model to do the same thing, and so I tried getting your program to work on my system. I ran into an error "Could not load the Qt platform plugin 'xcb'", among other similar errors from different package installations (to my understanding.)

If you are willing to, would you be able to provide a more comprehensive README on how to set up and get your model running?
I would appreciate it very much.

**Comment by zillur01** [comment]
Thanks for your interest. Can you please post a screenshot of the error here? You do not need to install anything for my model to work. Just install python packages for `yolov5`. Besides, please mention your system info like os, python version, cuda version, pytorch, etc. 
Also, can you please tell me which part of the readme file you difficult to understand? It would be helpful for me to update it.

**Comment by dejazzhands** [comment]
Sure! I think my current error has to do with a missing .pt file:
![image](https://github.com/zillur-av/wrong-way-vehicle-detection/assets/49312226/4a86e8b1-9fd7-4d87-9328-3cd2b96947b5)

OS: Windows 10 Home x64, Ubuntu Windows Subsystem for Linux
Python version: 3.8.10
Cuda version: No CUDA IDE installed, no GPU
Pytorch version: 2.0.1

**Comment by zillur01** [comment]
Yes, you did not download and put the `my_coco.pt` file in `./yolov5/my_coco.pt`. Please use the link mentioned in the readme file. Also, I have not tested it with Pytorch 2.0.1. In case it becomes an issue, use a conda environment on windows.

**Comment by dejazzhands** [comment]
this is the error i get after following the steps:
![image](https://github.com/zillur-av/wrong-way-vehicle-detection/assets/49312226/a3232f43-d5c3-4fe9-ac13-cd8b2c9e4f8b)

It appears to be the xcb error I was talking about. There's a guide on using opencv-python-headless instead, but I couldn't get it to work.
https://github.com/wkentaro/labelme/issues/842

UPDATE: This might be an issue with OpenCV on Linux.
